### PR TITLE
7. T5 - Prime Number checker

### PR DESCRIPTION
**Pull request by sathya-git-tech**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,20 @@
 ﻿namespace Training {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello World!");
+         Console.Write ("Enter a number:");
+         int n = int.Parse (Console.ReadLine ());
+         int result = 0;
+         for (int i = 2; i < n; i++) {
```
Comment (asfar-nazar) [inline Program.cs]: Do we need to check all numbers till n ?

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,20 @@
 ﻿namespace Training {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello World!");
+         Console.Write ("Enter a number:");
+         int n = int.Parse (Console.ReadLine ());
+         int result = 0;
```
Comment (asfar-nazar) [inline Program.cs]: Can you try to do this without using this local variable?

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (sathya-git-tech) [reply Program.cs]: yes snr

Comment (asfar-nazar) [reply Program.cs]: https://en.m.wikipedia.org/wiki/Primality_test

Comment (asfar-nazar) [reply Program.cs]: image

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+namespace Training {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello World!");
+         Console.Write ("Enter a number n:");
+         int n = int.Parse (Console.ReadLine ());
+         bool prime = true;
+         for (int i = 2; i < n/2; i++) {
+            if (n % i == 0) {
+               Console.WriteLine ("is not prime");
```
Comment (asfar-nazar) [inline Program.cs]: Console.WriteLine ($"{n} is not prime");

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

**Review by asfar-nazar — APPROVED** [review]

**Pull request closed without merging**